olsync/olsync.py

```diff
@@ -50,9 +52,14 @@ def main(ctx, local, remote, cookie_path, sync_path, olignore_path):
 
         overleaf_client = OverleafClient(store["cookie"], store["csrf"])
 
+        # FIXED: cd to sync_path
+        # current implemention list wrong file and get wrong project name
+        # and this implemention is more elegant without so many os.path.join
```

Comment:
Would remove "FIXED" comments and add a simple comment stating why directory changes here